cpp/leetcode/puzzles/dynamicprogramming/UniquePaths.cpp

```diff
@@ -0,0 +1,41 @@
+// UniquePaths.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//https://leetcode.com/problems/unique-paths/
+/*
+There is a robot on an m x n grid. The robot is initially located at the top-left corner (i.e., grid[0][0]). The robot tries to move to the bottom-right corner (i.e., grid[m - 1][n - 1]). The robot can only move either down or right at any point in time.
+
+Given the two integers m and n, return the number of possible unique paths that the robot can take to reach the bottom-right corner.
+*/
+
+#include <iostream>
+#include <vector>
+#include<assert.h>
+using namespace std;
+class Solution {
+public:
+
+    int uniquePaths(int rowsCount, int colsCount) {
+        vector<vector<int>> gridValues(rowsCount, vector<int>(colsCount, 1));
+        for (int row = 1; row < rowsCount; ++row) {
```

Comment:
compile your code with all warnings enabled (-Wall -Wextra ) and you will warnings for this code